messages.html

```diff
@@ -55,9 +52,105 @@
     </div>
   </nav>
 
+  <a class="waves-effect waves-light btn" href="#modal2" onclick="$('#modal1').modal('open');">New Message</a>
```

Comment:
we should probably keep this similar to material design standards -- a plus button in the corner. otherwise we could just follow the spec maybe?